themes/default/content/blog/pulumi-backstage-plugin/index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Backstage Plugin Now Available for Pulumi"
+date: 2023-10-06T15:00:48-04:00
+draft: false
+meta_desc: "Seamlessly integrate and manage your infrastructure with the official Pulumi Backstage plugin."
+meta_image: meta.png
+authors:
+    - engin-diri
+tags:
```

Comment:
@cnunciato Any other tags to add here?

Reply:
```suggestion
tags:
- backstage
- scaffolding
- internal-developer-portal
- idp
```

Reply:
Updated your suggestion a bit @dirien, but this seems okay to me!